### ISSUE TITLE
Document how to invoke these handlers directly for troubleshooting purposes

### ISSUE THREAD

**Issue by solarkennedy** [Yelp/sensu_handlers#39]
This is "just" cat'ing event json and piping it, but it isn't obvious.

Just an example taking some event data from the log and "replaying" it, pipe to a handler would be helpful for troubleshooting purposes.

**Comment by solarkennedy** [comment]
```
grep 'failed' /var/log/sensu/sensu-server.log  | tail -n 1 | jq .event > last_failed_event.json
cat last_failed_event.json | ./pagerduty.rb
```

**Comment by solarkennedy** [comment]
of course edit the json to tweak if necissary

**Comment by vulpine** [comment]
Fixed in e106709.
